src/Schema/Factories/FieldFactory.php

```diff
@@ -327,6 +332,23 @@ protected function handleArgDirectives(
         }
     }
 
+    protected function gatherValidationDirectives(Collection $directives)
+    {
+        // We only get the validator directives that are directly following on the latest validator
```

Comment:
Nicely done, even formatted in the canonical Laravel way. Love the attention to detail :)